docs/docs/api.rst

```diff
@@ -170,6 +172,7 @@ Those structure hold the state of the sampler.
 
   netket.sampler.SamplerState
   netket.sampler.MetropolisSamplerState
+  netket.sampler.ARDirectSamplerState
```

Comment:
Let's not put this here.
Metropolis and Sampler are there because people might extend them.
AR... I don't think that's the case (and regardless I don't want to draw attention to something internal )